stories/annotations/arrow.stories.tsx

```diff
@@ -17,6 +17,7 @@ export default {
     y2: 33,
     startPoint: 'none',
     endPoint: 'triangle',
+    color: 'red',
```

Comment:
Can you try to add another arrow in the same story with a different color? I am not sure that it will work if we define the shapes more than once with the same ids.

Reply:
i got a problem here and tried to fix it (u can check the last commit)

Reply:
now the color depend on the x1,y1.....
i don't know if it's the perfect solution

Reply:
It's probably good enough for now. We can revisit it later if it becomes an issue.

Reply:
okey perfect thank you